templates/agent.linkerd.yaml

```diff
@@ -0,0 +1,310 @@
+    process: rubrik.spark.blobstore.BlobstoreServiceMain
+  pushdown: []
+- node:
+    name: cloud_manager_bin
+    zone: .*
+    app: .*
+    pod: o365-cloud-manager
+    container: o365-cloud-manager
+    binary_name: /cloud-manager.bin
+    parent_process: containerd-shim
+    process: cloud-manager.bin
+  pushdown: []
+- node:
+    name: cloud_sql_proxy
+    zone: .*
+    app: rubrik-spark.*
+    pod: cloudsql-datastore-.*
+    container: cloudsql-proxy
+    binary_name: /cloud_sql_proxy
+    parent_process: containerd-shim
+    process: cloud_sql_proxy
+  pushdown: []
+- node:
+    name: exocompute_aws_linkerd2_proxy
+    zone: .*
+    app: rubrik-spark-exocompute
+    pod: exocompute.*aws.*
+    container: linkerd-proxy
+    binary_name: /usr/lib/linkerd/linkerd2-proxy
+    parent_process: containerd-shim
+    process: linkerd2-proxy
+  pushdown: []
+- node:
+    name: exocompute_bin
+    zone: .*
+    app: rubrik-spark-exocompute
+    pod: exocompute
+    container: exocompute
+    binary_name: /exocompute.bin
+    parent_process: containerd-shim
+    process: exocompute.bin
+  pushdown: []
+- node:
+    name: flagr
+    zone: .*
+    app: rubrik-spark.*
+    pod: .*flagr
+    container: .*flagr
+    binary_name: /go/src/github.com/checkr/flagr/flagr
+    parent_process: containerd-shim
+    process: flagr
+  pushdown: []
+- node:
+    name: goddess_updater
+    zone: .*
+    app: rubrik-spark.*
+    pod: goddess-updater
+    container: goddess-updater
+    binary_name: /goddess-updater.bin
+    parent_process: containerd-shim
+    process: goddess-updater.bin
+  pushdown: []
+- node:
+    name: kubelet
+    zone: .*
+    app: .*
+    binary_name: /home/kubernetes/bin/kubelet
+    parent_process: systemd
+    process: kubelet
+  pushdown: []
+- node:
+    name: linkerd2_proxy
+    zone: .*
+    app: .*
+    pod: .*
+    container: linkerd-proxy
+    binary_name: /usr/lib/linkerd/linkerd2-proxy
+    parent_process: containerd-shim
+    process: linkerd2-proxy
+  pushdown: []
+- node:
+    name: linkerd_await
+    zone: .*
+    app: .*
+    pod: .*
+    container: linkerd-proxy
+    binary_name: /usr/lib/linkerd/linkerd-await
+    parent_process: runc
+    process: linkerd-await
+  pushdown: []
+- node:
+    name: linkerd_controller
+    zone: .*
+    app: linkerd
+    pod: linkerd-.*
+    container: .*
+    binary_name: /controller
+    parent_process: containerd-shim
+    process: controller
+  pushdown: []
+- node:
+    name: linkerd_hearbeat_controller
+    zone: .*
+    app: linkerd
+    pod: linkerd-heartbeat
+    container: heartbeat
+    binary_name: /controller
+    parent_process: containerd-shim
+    process: controller
+  pushdown: []
+- node:
+    name: linkerd_policy_controller
+    zone: .*
+    app: linkerd
+    pod: linkerd-.*
+    container: policy
+    binary_name: /bin/linkerd-policy-controller
+    parent_process: containerd-shim
+    process: linkerd-policy-controller
+  pushdown: []
+- node:
```

Comment:
we dont need these

Reply:
which ones exactly are you referring to?